src/accordion/accordion.spec.ts

```diff
@@ -271,15 +272,78 @@ describe('ngb-accordion', () => {
          expectOpenPanels(fixture.nativeElement, [false, false, false]);
        });
      })));
+
+  it('should have default type and change when type is specified', async(inject([TestComponentBuilder], (tcb) => {
+       const testHtml = `
+      <ngb-accordion #acc="ngbAccordion">
+        <ngb-panel *ngFor="let panel of panels" [id]="panel.id" [disabled]="panel.disabled">
+          <template ngbPanelTitle>{{panel.title}}</template>       
+          <template ngbPanelContent>{{panel.content}}</template>        
+        </ngb-panel>        
+      </ngb-accordion>
+      <button *ngFor="let panel of panels" (click)="acc.toggle(panel.id)">Toggle the panel {{ panel.id }}</button>
+    `;
+       tcb.overrideTemplate(TestComponent, testHtml).createAsync(TestComponent).then((fixture) => {
+         fixture.detectChanges();
+
+         let el = fixture.nativeElement.querySelectorAll('.card-header');
+         expect(el[0]).toHaveCssClass('card-primary');
+         expect(el[1]).toHaveCssClass('card-primary');
+         expect(el[2]).toHaveCssClass('card-primary');
```

Comment:
This is not correct! We shouldn't get any class other than `card-header` (and eventually `active`) by default. Run the demo and see how it looks....